src/_pytest/helpconfig.py

```diff
@@ -12,6 +12,11 @@
 from _pytest.config import PrintHelp
 from _pytest.config.argparsing import Parser
 
+try:
```

Comment:
if we repeat that all over the place it might make sense to have a single module that exposes the api we need to have a optional dep

Reply:
You means copy `shtab` to this package wholly not create a fake `_shtab.py`? If so, it must be maintained together and may be not the thought of most people.

Reply:
You means copy `shtab` to this package wholly not create a fake `_shtab.py`? If so, it must be maintained together and may be not the thought of most people.